includes/class-wc-siftscience-admin.php

```diff
@@ -369,8 +368,6 @@ private function output_settings_debug() {
 				exit;
 			}
 
-			$logs = 'none';
-
 			$GLOBALS['hide_save_button'] = true;
 
 			if ( file_exists( $log_file ) ) {
```

Comment:
Ah! I know what the problem is, and there's an easier way to solve this.

The problem is that the file can exist, but it's empty.